### ISSUE TITLE
Side Bar icon does not display based on order of panels

### ISSUE THREAD

**Issue by wsmelton** [microsoft/vscode-pull-request-github#2773]
> Not sure if this may not be an upstream issue with GitHub?

- Extension version: v0.27.0
- VSCode Version:
```
Version: 1.58.0-insider (system setup)
Commit: 012b14606f75ec4fed90d7ab4154f63dffab9503
Date: 2021-06-11T05:14:11.473Z
Electron: 12.0.9
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19042
```
- OS: Windows 10 Enterprise 20H2

Steps to Reproduce:

1. Open Side Bar - GitHub
2. Reorder the panels, moving Issues to the top and Pull Requests to the bottom

![image](https://user-images.githubusercontent.com/11204251/121749501-63204680-cad0-11eb-9c30-bbafa0a6c5a8.png)

vs with Pull Request first

![image](https://user-images.githubusercontent.com/11204251/121749463-513ea380-cad0-11eb-910c-77b3db8e74c5.png)

**Comment by alexr00** [comment]
@wsmelton I'm not sure how you got into this state. 

tl;dr: You can fix this by running the command. Views: Reset View Locations. This will reset all your view locations though, so if you have some moved around intentionally you'll lose that too. 

But here's what's happening:

This extension has two **views**: Pull Requests and Issues.
This extension has a **view container**: GitHub. This is what shows in the activity bar (usually).
The two views are added to the view container, which has a "github" icon normally. You are not seeing that icon.
If you drag the two views out of the original "GitHub" view and on to the sidebar, a new view container is created just for that view. You can then drag other views into that new view container. The view container will get the icon associated with the first view in in it.
What it looks like has happened is that you dragged the "Pull Requests" view into a new view container, then you dragged the "issues" view into that same new view container. The abandoned the old "GitHub" view container, causing it to hide. The new view container now takes it's icon from the first view in it.
